scipy/stats/_stats_py.py

```diff
@@ -7617,11 +7624,23 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', method='auto'):
 
     Returns
     -------
-    statistic : float
-        KS test statistic, either D, D+ or D- (depending on the value
-        of 'alternative')
-    pvalue : float
-        One-tailed or two-tailed p-value.
+    res: KstestResult
+        An object containing attributes:
+
+        statistic : float
+            KS test statistic, either D, D+ or D- (depending on the value
+            of 'alternative')
+        pvalue : float
+            One-tailed or two-tailed p-value.
+        statistic_location: float
+            Value of the stochastic variable closest to the KS statistic. By
+            definition, the distance between the theoretical and sampled
+            distributions at this value is largest.
```

Comment:
"closest to"? 
The KS statistics is the difference between the theoretical and empirical CDFs _at_ this location, no?

Reply:
It's mostly wording to ward off people who are worried about aliasing. Happy to rephrase more directly.